src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/ImportOrderCheck.java

```diff
@@ -140,6 +140,30 @@
  * </li>
  * </ul>
  * <p>
+ * To configure the check:
+ * </p>
+ * <pre>
+ * &lt;module name="ImportOrder"/&gt;
+ * </pre>
+ * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * import java.io.IOException; // OK
+ * import java.net.URL; // OK
+ *
+ * import java.io.IOException; // violation, extra separation before 'java.io.IOException'.
+ *                             // and wrong order for 'java.io.IOException' import.
+ * import javax.swing.JComponent; // violation, extra separation before 'javax.net.ssl.SSLContext'
```

Comment:
same here

Reply:
here 157 line 
`extra separation due to above comment` is it okay?

Reply:
yes

Reply:
done